<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="all_products_list">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>451</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>All avaliable products</string>
    </property>
   </widget>
   <widget class="QPushButton" name="productDocx">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export info about all products *.docx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="productExcel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export info about all products *.excel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search product by name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="enter_prod_name">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchProdBtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QListView" name="search_product_list">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>100</y>
      <width>351</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add new product</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prod_name">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prod_unit">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter measurment unit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sell_price">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>120</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter sell price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="purchase_price">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>160</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter purchase price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_prod">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="manipulate_name">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>280</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter product name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manipulate_prod">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Manipulate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Update list</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
